cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -829,12 +830,6 @@ bool SubstraitToVeloxPlanValidator::validate(const ::substrait::FilterRel& filte
     logValidateMsg("native validation failed due to: Validation failed for input types in FilterRel.");
     return false;
   }
-  for (const auto& type : types) {
```

Comment:
Removed this completely to check if all UTs passes, as there are specific validation for timestamp cases. If some case are missing, I can add better validation check which can validate if predicate is on timestamp column.